### ISSUE TITLE
Edit front matter in UI

### ISSUE THREAD

**Issue by phae** [codeforamerica/ceviche-cms#69]
Front-matter in the actual content files should probably be hidden, but shown as a selection of dropdowns/inputs based on config values etc. so that they can't get screwed up. :)

**Comment by migurski** [comment]
Can you give some examples of front-matter things that should be editable, other than titles and non-English content?

**Comment by phae** [comment]
Yeah, it'll be like setting which department owns that content, so that the page will know to pull in that departments content info block for the layout. That sort of thing, where there needs to be content cross-links and we don't want users screwing it up/adding new ones when it's being defined in the core config file.
